### PR TITLE
Chef 13 compatibility fixes

### PR DESCRIPTION
**Pull request by jcorme**

### CODE REVIEW

**Comment by jcorme** [comment]
[Chef 13: osl-nginx](https://trello.com/c/5kSJWz7z/932-chef-13-osl-nginx)

**Comment by TravisWhitehead** [comment]
Oops...

**Comment by ramereth** [comment]
@jcorme can you please switch this back to the [nginx cookbook](https://supermarket.chef.io/cookbooks/chef_nginx)? It seems they finally fixed the upstream cookbook and no longer maintain this fork.

File: metadata.rb [hunk 1]
```diff
@@ -3,15 +3,16 @@
...
+chef_version     '>= 12.18' if respond_to?(:chef_version)
 description      'Installs/Configures osl-nginx'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '2.1.2'
 
 depends          'certificate'
 depends          'firewall'
 depends          'logrotate'
-depends          'chef_nginx', '~> 6.2.0'
+depends          'nginx'
```
Comment (ramereth) [inline metadata.rb]: Please add a pessimistic lock using the most current version for this since this is a wrapper cookbook for this.

**Comment by jcorme** [comment]
!bump major *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v3.0.0.  Have a nice day!

**Pull request merged**